compiler/optimizer/pom.xml

```diff
@@ -47,6 +47,11 @@ under the License.
       <artifactId>nemo-conf</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.nemo</groupId>
```

Comment:
compile-time optimizer requires runtime info? 

or does this should be runtime optimizer?

Reply:
Optimizations are required for the metric data which are collected during runtime. I've exposed this bit so that it could be accessed.